### PR TITLE
Created Layout and homepage

### PR DESCRIPTION
**Pull request by aryanandarnav13**

### CODE REVIEW

File: home.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Home | Store</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <script src="https://kit.fontawesome.com/a076d05399.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="home.css" />
+    <!-- custom css -->
```
Comment (keshavkumar-in) [inline home.html:11]: You have commented custom css below the CSS you called

File: home.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+      </div>
+      <div class="nav-btn">
+        <label for="nav-check">
+          <span></span>
+          <span></span>
+          <span></span>
+        </label>
+      </div>
+      <div class="nav-links">
+        <a href="#" target="_blank">Home</a>
```
Comment (keshavkumar-in) [inline home.html:31]: `target blank` is used to open the links in another window. it's not required when you are navigating inside the site

**Review by keshavkumar-in — APPROVED** [review]
Added just few suggestions, other than that looks good to me.

File: home.html (hunk 1 above)
Comment (aryanandarnav13) [reply home.html:11]: Yaah noted, Thanks Keshav

File: home.html (hunk 2 above)
Comment (aryanandarnav13) [reply home.html:31]: okay Keshav, Thanks I didn't know this

**Review by sks-21 — APPROVED** [review]
The code looks nice. Good to merge

**Pull request merged**